api/cosmos/app/runtime/v1alpha1/module.pulsar.go

```diff
@@ -2217,7 +2217,7 @@ type Module struct {
 	SkipStoreKeys []string `protobuf:"bytes,11,rep,name=skip_store_keys,json=skipStoreKeys,proto3" json:"skip_store_keys,omitempty"`
 	// order_migrations defines the order in which module migrations are performed.
 	// If this is left empty, it uses the default migration order.
-	// https://pkg.go.dev/github.com/cosmos/cosmos-sdk@v0.47.0-alpha2/types/module#DefaultMigrationsOrder
```

Comment:
we shouldn't modify this directly - should modify the proto file that owns this doc

Reply:
Ah I had it the wrong way around. Apologies! Fix pushed.